cardano-node/cardano-node.cabal

```diff
@@ -38,6 +38,7 @@ common project-config
                         -Wpartial-fields
                         -Wredundant-constraints
                         -Wunused-packages
+                        -fprof-auto
```

Comment:
This introduces overhead in the runtime - do we want that to be merged into `master`?

Reply:
I was not aware, that this actually gives a runtime overhead. I will remove it everywhere.

Reply:
Sorry, fixed it. The tracing tools are partly included in the node as libraries, so I care about them the same way

Reply:
Thanks!